core/entrypoint/2-copy-config.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+if [ -z "$CHORUS_USER" ]; then
+  echo "CHORUS_USER is not set. Exiting."
+  exit 1
+fi
+
+if [ -z "$CONFIG_ARRAY" ]; then
+  echo "CONFIG_ARRAY is not set. Exiting."
+  exit 0
+fi
```

Comment:
if you flip those two conditions, it exits with a success. Which one is the most important?

Reply:
The user needs to be set , the config array might be empty, or not.
The idea was just that this script shouldn't crash if there is no config to copy in case we make a full bundle with core scripts instead of just copying the scripts needed like we do now (less error on the user side)

Reply:
then flip them around; if the USER needs to be set, I don't think this is the right place to perform this check.

Reply:
Hum all in all we can remove the user part , this was more of a security for me when i was testing. This is indeed not the place to check that.